interpretation/ww_head.py

```diff
@@ -90,17 +96,24 @@ def process(message, isGameMaster = False):
             user_table = check.users(message)
             identities = Mailbox()
 
+            if user_table == False:
+                return [Mailbox().respond("**ERROR:** No user provided!")]
+            
             for user in user_table:
                 emoji = db_get(user,'emoji')
                 role = db_get(user,'role')
-                msg = "{} - <@{}> has the role of the `{}`!".format(emoji,user,role)
-                identities.spam(msg)
+                if emoji == None or role == None:
+                    identities.spam("**ERROR:** Could not find user <@{}> in database.".format(user))
+                else:
+                    msg = "{} - <@{}> has the role of the `{}`!".format(emoji,user,role)
+                    identities.spam(msg)
 
             return [identities]
 
         if is_command(message,['whois'],True):
             msg = "**Usage:** `" + prefix + "whois <user1> <user2> ...`\n\n"
             msg += "Example: `" + prefix + "whois @Randium#6521`\nGame Master only command"
+            return [Mailbox().respond(msg,True)
```

Comment:
Whoops! Could you add a ] at the end of this line, please? My bad. 😬

Reply:
Gee, just do it yourself :P here, I'll do it for you.